contributed_definitions/NXcg_cylinder_set.nxdl.xml

```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0" ?>
+<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl"?>
+<definition category="base" extends="NXobject" name="NXcg_cylinder_set" type="group" xmlns="http://definition.nexusformat.org/nxdl/3.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd">
+    <symbols>
+        <doc>
+             The symbols used in the schema to specify e.g. dimensions of arrays.
+        </doc>
+        <symbol name="c">
+            <doc>
+                 The cardinality of the set, i.e. the number of cylinders or cones.
+            </doc>
+        </symbol>
+    </symbols>
+    <doc>
+         Set of cylinders or (truncated cones) in Euclidean space.
```

Comment:
the choice of the name cg_... shall be clear from the text

Reply:
done